core/templates/calendar.html

```diff
@@ -102,4 +102,5 @@
   window.allEvents = <%- JSON.stringify(allCalendarEvents) %>;
   window.userName = "<%- user %>";
 </script>
-<script src="/js/cal.js"></script>
+<script src="/socket.io/socket.io.js"></script>
```

Comment:
this is included in the layout template, so you can safely remove this line here